### ISSUE TITLE
Resistance to induced hyperthermia

### ISSUE THREAD

**Issue by mnk77** [obophenotype/mammalian-phenotype-ontology#2630]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
resistance to induced hyperthermia

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
Reduced increase in body temperature following treatment to induced hyperthermia (eg. isopropanol).

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.
PMID 24531262

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
abnormal adaptive thermogenesis
MP:0011019

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
MGI:mperry

**Comment by sbello** [comment]
we already have terms related to hyperthermia (MP:0010165 under behavior, MP:0020214 under physiological sensitivity to xenobiotic) need to figure out how this relates to these

**Comment by sbello** [comment]
Does MP:0010165 really belong under behavior? This is a physiological response not really a behavioral one

**Comment by sbello** [comment]
added term 
decreased susceptibility to xenobiotic induced hyperthermia MP:0020522
